cmd/pd-server/main.go

```diff
@@ -108,39 +109,26 @@ func createServerWrapper(cmd *cobra.Command, args []string) {
 	flagSet := cmd.Flags()
 	flagSet.Parse(args)
 	err := cfg.Parse(flagSet)
+	defer logutil.LogPanic()
+
 	if err != nil {
 		cmd.Println(err)
 		return
 	}
 
-	printVersion, err := flagSet.GetBool("version")
-	if err != nil {
+	if printVersion, err := flagSet.GetBool("version"); err != nil {
 		cmd.Println(err)
 		return
-	}
-	if printVersion {
-		server.PrintPDInfo()
+	} else if printVersion {
+		versioninfo.Print()
 		exit(0)
 	}
 
-	defer logutil.LogPanic()
-
-	switch errors.Cause(err) {
```

Comment:
This is dead code, because err will never be nil here as it has been checked above. This becomes dirty code after several refactors that we did.